src/app/components/neo-button/neo-button/_button-base.scss

```diff
@@ -1,6 +1,6 @@
 @mixin neo-button-base {
   $icon-size: 20px;
-  $button-size: 1.6rem !default;
+  $button-size: 40px !important;
```

Comment:
Could we avoid use important at framework styles?

Reply:
done